### PR TITLE
enh(epp): applications-rubrik-restapi - update proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 1]
```diff
@@ -5,69 +5,126 @@ title: Rubrik Rest API
...
-## Contenu du connecteur de supervision
+### Modèles
 
-### Objets supervisés
+Le connecteur de supervision **Rubrik Rest API** apporte un modèle d'hôte :
 
-Le connecteur de supervision Centreon Rubrik permet (par l'interrogation de l'API Rest) de superviser 
-le statut et les performances Clusters et ses Noeuds. Cela inclut notamment les composants 
-disques, les tâches, etc.
+* **App-Rubrik-Restapi**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:14]:
```suggestion
* **App-Rubrik-Restapi-custom**
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 2]
```diff
@@ -5,69 +5,126 @@ title: Rubrik Rest API
...
 
-### Métriques collectées
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
 
 <Tabs groupId="sync">
-<TabItem value="Disk" label="Disk">
+<TabItem value="App-Rubrik-Restapi" label="App-Rubrik-Restapi">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:20]:
```suggestion
<TabItem value="App-Rubrik-Restapi-custom" label="App-Rubrik-Restapi-custom">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 3]
```diff
@@ -5,69 +5,126 @@ title: Rubrik Rest API
...
+| Alias      | Modèle de service             | Description                            |
+|:-----------|:------------------------------|:---------------------------------------|
+| Cluster    | App-Rubrik-Cluster-Restapi    | Contrôle le cluster                    |
+| Compliance | App-Rubrik-Compliance-Restapi | Contrôle la conformité des sauvegardes |
+| Disks      | App-Rubrik-Disks-Restapi      | Contrôle les disques                   |
+| Nodes      | App-Rubrik-Nodes-Restapi      | Contrôle les noeuds du cluster         |
+| Storage    | App-Rubrik-Storage-Restapi    | Contrôle le système de stockage        |
+| Tasks      | App-Rubrik-Tasks-Restapi      | Contrôle les tâches                    |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Rubrik-Restapi** est utilisé.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:31]:
```suggestion
> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Rubrik-Restapi-custom** est utilisé.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 4]
```diff
@@ -86,61 +143,239 @@ _Note : les paramètres internes de l'API peuvent changer entre les versions du
...
+
+### Utiliser un modèle d'hôte issu du connecteur
+
+1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
+3. Appliquez le modèle d'hôte **App-Rubrik-Restapi-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro                 | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:----------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:242]: are there any mandatory ones?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 5]
```diff
@@ -86,61 +143,239 @@ _Note : les paramètres internes de l'API peuvent changer entre les versions du
...
+| RUBRIKAPIPASSWORD     | API password                                                                                          |                   |             |
+| RUBRIKAPIPORT         | Port used                                                                                             | 443               |             |
+| RUBRIKAPIPROTO        | Specify https if needed                                                                               | https             |             |
+| RUBRIKAPIUSERNAME     | API username                                                                                          |                   |             |
+| RUBRIKAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:247]:
```suggestion
| RUBRIKAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 6]
```diff
@@ -86,61 +143,239 @@ _Note : les paramètres internes de l'API peuvent changer entre les versions du
...
 
-### Comment puis-je tester le Plugin et que signifient les options des commandes ?
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
 
-Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande
-depuis un collecteur Centreon en vous connectant avec l'utilisateur *centreon-engine*:
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser un serveur en utilisant une commande
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:377]:
```suggestion
que le connecteur arrive bien à superviser la ressource en utilisant une commande
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**